test/benchmarks/taylor.c

```diff
@@ -0,0 +1,19 @@
+#define WASM_EXPORT __attribute__((visibility("default")))
+
+// Pretty bad Pi approximation using Taylor series
+WASM_EXPORT unsigned long long taylor(unsigned n)
+{
+    double sum = 1.0;
+    int sign = -1;
+    
+    for (unsigned i = 1; i < n; i++) {
+       sum += sign / (2.0 * i + 1.0);
+       sign = -sign;
+    }
+
+    // This is the result in double precision
+    sum = 4.0 * sum;
+
+    // Display all 16 digits of double precision as a 64-bit integer
+    return sum * 10000000000000000ULL;
```

Comment:
First I returned f64. Supporting that in wasm_engine/fizzy_engine is trivial, but changing the fizzy-bench parser seemed like a larger task so decided for this "workaround".